tests/test_conftest.py

```diff
@@ -0,0 +1,5 @@
+# Copyright (C) 2024 by Siisurit e.U., Austria.
+# All rights reserved. Distributed under the MIT License.
+
+from dotenv import load_dotenv
+load_dotenv()
```

Comment:
Can you explain why did you create a new test file that doesn't align with any module and that does nothing but load the environment variables?

This was not part of the requirements.

You mentioned in the PR "Ensure everything works by testing the environment", but nothing is being actually tested here. And no environment-variable-dependent test is needed to complete the requirements of this issue.